content/references/protocol-reference/transactions/transaction-types/nftokenmint.md

```diff
@@ -10,9 +10,9 @@ status: not_enabled
 [[Source]](https://github.com/ripple/rippled/blob/xls20/src/ripple/app/tx/impl/NFTokenMint.cpp)
 {% include '_snippets/nfts-disclaimer.md' %}
 
-The `NFTokenMint` transaction creates an non-fungible token and adds it to the relevant [NFTokenPage object][] of the `Minter` as an [NFToken][] object. A required parameter to this transaction is the `Token` field specifying the actual token. This transaction is the only opportunity the `Minter` has to specify any token fields that are defined as immutable (for example, the `TokenFlags`).
+The `NFTokenMint` transaction creates a NFToken and adds it to the relevant [NFTokenPage object][] of the `NFTokenMinter` as a [NFToken][] object. A required parameter to this transaction is the `Token` field specifying the actual token. This transaction is the only opportunity the `NFTokenMinter` has to specify any token fields that are defined as immutable (for example, the `Flags`).
```

Comment:
My sensibilities would be inclined to say "an NFToken" rather than "a NFToken".  But you are better an English than I am.

Reply:
We'll see how it shakes out. Some say en-eff-tees, some say nifties and niftokens.